File: patterns/c-patterns/pattern86.c
// Pattern 86: ...

// *
// * *
// 123
// * *
// *

#include <stdio.h>

int main()
{
    int n = 5;
    int i, j;
    int x = 1;

    for (i = 1; i <= n; i++)
    {
        for (j = 1; j <= x; j++)
        {
            if (i != n / 2 + 1)
            {
                printf("* ");
            }
            else
            {
                printf("%d", j);
            }
        }
        if (i <= n / 2)
            x++;
        else
            x--;
        printf("\n");
    }
    return 0;
}

File: patterns/python-patterns/pattern86.py
# Pattern 86: ...

# *
# * *
# 123
# * *
# *

#include <stdio.h>

n = 5
x = 1

for i in range(1, n+1):
    for j in range(1, x+1):
        if i != n//2 + 1:
            print("* ", end="")
        else:
            print(j, end=" ")
    if i <= n//2:
        x += 1
    else:
        x -= 1
    print()
